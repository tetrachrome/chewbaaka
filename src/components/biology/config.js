/**
 * biology/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 15, 2020
 * Updated  : Jul 18, 2020
 */

export const config = {
  pageProps: {
    coverImage: "",
    title: "Biology",
    subtitle: "Subtitle TBD ...",
    pageMenuItems: [
      {
        title: "Lifecycle",
        tocImageFilename: "biology_page_toc_menu_img_01.jpg"
      },
      {
        title: "Physiology",
        tocImageFilename: "biology_page_toc_menu_img_02.jpg"
      }
    ],
  },
  contentPageIntro: {
    title: "Cheetah Biology",
    content: "The cheetah is a truly unique animal. Being the world's fastest land animal, it is built for speed. " +
            "An acceleration from 0 to 84 km/hr in just 3 seconds, with a full speed of 110 km/hr, means that the " +
            "cheetah can out-perform a sports car. Cheetahs are markedly different in both anatomy and behavior " +
            "from the other 36 species of cats. They are the only species in their genus. " +
            "They have evolved for speed versus power and aggression. Their bodies are lightweight in comparing with " +
            "the build of other big cats. They rely on their speed and skills for survival. The genus name, Acinonyx, " +
            "is interpreted as “non-moving claws”, referring to the semi-retractable claws. " +
            "The species name, “jubatus”, means “maned”, referring to the mantle on a young cheetah’s back. " +
            "The English word, “cheetah”, comes from the Hindu word “chita” meaning the “spotted one”."
  },
  contentPageSections: {
    section_Lifecycle: {
      intro: {
        title: "Lifecycle",
        content: "A cheetah goes through 4 stages of life in its life."
      },
      subsections: {
        subsection_Lifecycle_Stage_1: {
          title: "Stage 1",
          contents: {
            paragraph_Lifecycle_Stage_1_01:
              "The female’s gestation period lasts 90 to 95 days. " +
              "This means she is pregnant for about three months.",
            paragraph_Lifecycle_Stage_1_02:
              "Shortly before she is ready to give birth, the mother " +
              "makes a den in a quiet hidden spot. She chooses her location " +
              "in the tall grass think underbrush or near a clump of rocks.",
            paragraph_Lifecycle_Stage_1_03:
              "A cheetah gives birth to an average of five to six cubs. " +
              "Each cub weights between 250 to 425 grams. Cubs are born " +
              "completely helpless and with their eyes closed, but they " +
              "developer rapidly. Scent and touch are used to find their " +
              "mother’s nipples to suckle.",
            paragraph_Lifecycle_Stage_1_04:
              "They start crawling around the nest area at four to ten days "+
              "when their eyes begin to open. At three weeks their teeth break " +
              "through the gums.",
            paragraph_Lifecycle_Stage_1_05:
              "A mother cheetah will frequently move her litter. This prevents " +
              "a build-up of scent of the den site, which may lead other predators " +
              "to the cubs. She carries very young cubs in her jaws.",
            paragraph_Lifecycle_Stage_1_06:
              "The cubs are very vulnerable to lions, hyenas and other predators when "+
              "the female leaves them alone. When hunting she may be away for up to 48 hours. " +
              "In Tanzania’s Serengeti National Park, 90% of all cubs do not reach the age " +
              "of 3 months! Other causes of death are abandonment when prey is scarce, " +
              "exposure due to low temperates or grass fires."
          }
        },
        subsection_Lifecycle_Stage_2: {
          title: "Stage 2",
          contents: {
            paragraph_Lifecycle_Stage_2_01:
              "At 1.5 to 2 months of age, the cubs leave the safety of the den " +
              "to accompany their mother. They are very vulnerable as they are not " +
              "able to defend themselves.",
            paragraph_Lifecycle_Stage_2_02:
              "They stop drinking their mother’s milk at 3 to 4 months of age. " +
              "They start eating meat and learning to hunt. The games they play and the " +
              "experiences they have during this stage will teach them skills needed to survive on their own."
          }
        },
        subsection_Lifecycle_Stage_3: {
          title: "Stage 3",
          contents: {
            paragraph_Lifecycle_Stage_3_01:
              "Stage three begins at the age 18 to 22 months when the cubs have grown " +
              "to sub-adults and leave their mother. The sub-adults will remain together for " +
              "up to six more months. At first, their success rate at capturing prey is poor.",
            paragraph_Lifecycle_Stage_3_02:
              "Cheetahs are diurnal, hunting mornings and early evenings. They rely on their " +
              "sight to find prey. They spend most of the day resting under shady trees or on " +
              "termite mounds. Night hunting is only done during a bright moon."
          }
        },
        subsection_Lifecycle_Stage_4: {
          title: "Stage 4",
          contents: {
            paragraph_Lifecycle_Stage_4_01:
              "In Stage 4, cheetahs become sexually mature. Although they are mature at 16 to 18 months, " +
              "most do not breed until they are three to five years old.",
            paragraph_Lifecycle_Stage_4_02:
              "At 20 - 30 months of age, females leave their litter-mates to find suitable mates " +
              "and start their own families. They raise their families on their own without the help of the male.",
            paragraph_Lifecycle_Stage_4_03:
              "Males usually do not breed until they are 4 to 5 years of age, and dominant in a territory. "+
              'They live alone or brothers farm permanent groups called "coalitions". '+
              "These groups stay together for life, claim territories, hunt and find mates together."
          }
        },
        subsection_LivingFastDyingYoung: {
          title: "Living Fast - Dying Young",
          contents: {
            part1: {
              title: "Male coalitions",
              content: {
                paragraph_01:
                  "A coalition consists of a male siblings of the same litter or young unrelated " +
                  "males that have joined together. A hierarchy develops among the males within a coalition.",
                paragraph_02:
                  "Young males are usually chased away from their birth range by dominant breeding males. " +
                  "They may establish home ranges more than 100 km away. A coalition is more successful in acquiring " +
                  "and holding territories and in defending kills than single males. " +
                  "This competition can result in mortality among males."
              }
            },
            part2: {
              title: "Mortality",
              content: {
                paragraph_01:
                  "The cheetah’s lifespan is poorly documented in the wild. A radio collared cheetah lived to be " +
                  "almost 7 years of age in Tanzania’s Serengeti National Park and once of CCF’s radio collared " +
                  "cheetahs in Namibia lived for over 10 years.",
                paragraph_02:
                  "Adult mortality is one of the most significant limiting factors for cheetah population growth and survival. " +
                  "Poaching, competition with large predators and farmers, and loss of habitat and prey are factors " +
                  "attributing to early death.",
                paragraph_03:
                  "Though cub deaths are high, cheetahs have evolved to reproduce rapidly in response to this mortality."
              }
            }
          }
        },
        subsection_FindingMate: {
          title: "Finding a Mate",
          contents: {
            paragraph_FindingMate_01:
              "The range of a female offspring may partially overlap that of her mother. " +
              "Namibian cheetahs are more social than those reported in other countries. " +
              "Females are often seen with multiple adults and cubs of varying ages.",
            paragraph_FindingMate_02:
              "Female cheetahs are polyoestrous, which means there is no regular breeding season. " +
              "If not bred, females come into heat (estrus) several times a year. Estrus means they are ready to breed. " +
              "If cubs are lost to predators females soon come into estrus again.",
            paragraph_FindingMate_03:
              "Smell, sound, and behavioral stimuli attract males to females. Female cheetahs leave a scent trail " +
              "by releasing sex hormones in urine and feces. They mark trees and bushes. " +
              "This behavior increases during courtship.",
            paragraph_FindingMate_04:
              "When courtship takes place, males will follow females closely and mock fighting may be observed.",
            paragraph_FindingMate_05:
              "When a female is ready to mate, she adopts a receptive posture. The male mounts the female, " +
              "bites the back of her neck, and breeding takes place. When the male dismounts the female she " +
              "rolls over on her back and swats at him.",
            paragraph_FindingMate_06:
              "Mating will take place for one to several days and ends when the make loses interest in the female and leaves. " +
              "Males do not help raise the cubs."
          }
        }
      },
    },
    section_Physiology: {
      intro: {
        title: "Physiology",
        content: "TBD ..."
      },
      subsections: {
        subsection_Anatomy: {
          title: "Anatomy"
        },
        subsection_FeetAndClaws: {
          title: "Feet and Claws",
          contents: {
            part_Foot: {
              title: "Foot",
              content: {
                paragraph_01:
                  "Cheetah’s foot pads are hard and less rounded than the other cats. " +
                  "The pad function like tire threads providing them with increased traction in fast, sharp turns.",
              }
            },
            part_Claw: {
              title: "Claw",
              content: {
                paragraph_01:
                  "The short blunt claws work like the cleats on a track shoe. They grip the ground for " +
                  "traction when running and help increase speed.",
                paragraph_02:
                  "Cheetah’s claws are semi-retractable, meaning they do not completely retract like the claws " +
                  "of other cats. The foot structure of the cheetah is very dog-like.",
              }
            },
            part_Dewclaw: {
              title: "Dewclaw",
              content: {
                paragraph_FeetAndClaws_04:
                  "The dewclaws of the cheetah are located on the upper inside area of the front foot. " +
                  "These are sharp and frequently used to hook and hold prey."
              }
            }
          }
        },
        subsection_Skull: {
          title: "Skull",
          contents: {
            paragraph_Skull_01:
              "All predators have specialized features for catching and kill prey. " +
              "Predators have large forward facing eye sockets, and large ear cavities " +
              "that provides space for specialized senses. Although there are differences " +
              "in skull proportions, carnivore skulls are quite similar.",
            paragraph_Skull_02:
              "The cat skulls are different from other carnivore species, as they have a " +
              "flat nose and enlarge nasal cavity. There is a large area for the strong jaw muscles.",
            paragraph_Skull_03:
              "The cheetah has some features that are distinct when compared to the skulls of other cats."
          }
        },
        subsection_SpotsAndStripes: {
          title: "Spots and Stripes",
          contents: {
            paragraph_SpotsAndStripes_01:
              "Adult cheetahs are easily distinguished from other cats by their coat patterns. " +
              "The color and spots are a form of camouflage. This helps cheetahs hunt prey and " +
              "hide from other predators.",
            paragraph_SpotsAndStripes_02:
              "Distinctive black tear stripes run from the eyes to the mouth. The stripes are thought " +
              "to protect the eyes from the sun’s glare. It is believed they have the same function " +
              "as a rifle scope, helping cheetahs focus on their prey.",
            paragraph_SpotsAndStripes_03:
              "Until three months of age, cheetah cubs have a thick-silvery grey mantle down their back. " +
              "The mantle helps camouflage the cubs by blending them into the shadows and grass. " +
              "It also provides protection from sun and rain.",
            paragraph_SpotsAndStripes_04:
              "With their mantle, cubs look like an aggressive animal called a honey badger. " +
              "This may deter predators such as lions, hyenas, and eagles from attempting to kill them. " +
              "This is known as “mimicry”."
          }
        },
        subsection_HearAndLung: {
          title: "Heart and Lung",
          contents: {
            part1: {
              content: {
                paragraph_01:
                  "The cheetah has a large strong heart that rapidly pumps large amounts of oxygenated " +
                  "blood from the lungs to the muscles to keep them supplied with energy while running.",
                paragraph_02:
                  "Large lungs provide adequate oxygen for a cheetah’s increased energy needs while pursuing its prey. " +
                  "The cheetah’s respiratory rate climbs from 60 to 150 breathes per minute, nearly twice as fast as humans.",
                paragraph_03:
                  "Cheetahs have enlarged nostrils and sinuses allowing an increase in air flow to the lungs.",
                paragraph_04:
                  "High speeds can, however, only be maintained for 400 - 800 meters before exhaustion sets in " +
                  "and the body risks overheating."
              }
            },
            part2: {
              content: {
                paragraph_01:
                  "A cheetah running the 249km from Otjiwarongo to Windhoek would need to stop to rest more than 311 times.",
                paragraph_02:
                  "If it could run there without resting, it would take 2 hours and 26 minutes at 110km/hr."
              }
            }
          }
        },
        subsection_BodyAndBone: {
          title: "Body and Bone",
          contents: {
            part_Body: {
              title: "Body",
              contents: {
                paragraph_01:
                  "The cheetah’s body is narrow and lightweight with long, slender limbs. " +
                  "Specialized muscles allow a greater swing to the limbs, increasing acceleration.",
              }
            },
            part_Tail: {
              title: "Tail",
              contents: {
                paragraph_01:
                  "The cheetah’s long muscular tail works as a rudder, stabilizing and acting as a counter " +
                  "balance to its body weight. This allows sudden sharp turns during high speed chases.",
                paragraph_02:
                  "The tail is thought to be a signaling device, helping young cheetahs follow their mother in tall grass. " +
                  "The tip may be black or white in color."
              }
            },
            part_Bone: {
              title: "Bone",
              contents: {
                paragraph_01:
                  "The extreme flexibility of the cheetah’s spine is unique. This allows more extension during running, " +
                  "thus making both its stride length and speed possible.",
                paragraph_02:
                  "If the spine were stiff and the pectoral and pelvic girdles were firmly attached, the cheetah would not be " +
                  "able to reach 100 km / hr.",
                paragraph_03:
                  "The hips (pelvic girdle) pivot to increase the cheetah’s stride length. This allows the front and " +
                  "rear legs to stretch further apart when the body is fully extended. The hips and shoulders move " +
                  "closer together when the feet come under its body.",
                paragraph_04:
                  "The shoulder bone (pectoral girdle) does not attach to the collarbone, thus allowing the shoulders to move freely. " +
                  "This increases the length of the stride."
              }
            }
          }
        },
        subsection_OpenWide: {
          title: "Open Wide",
          contents: {
            part1: {
              paragraph_OpenWide_01:
                "The cheetah takes smaller prey than other similar sized cats. Their strong jaws " +
                "lock around the throat of the prey in what is called the “strangulation hold”, "+
                "which can last up to 20 minutes.",
              paragraph_OpenWide_02:
                "Cheetahs’ jaws are not as powerful as that of lions or leopards. Cheetahs have " +
                "comparatively shorted canine teeth. In all cats, powerful muscles move the jaw up " +
                "and down and provide vice-like strength for gripping prey.",
            },
            part2: {
              paragraph_OpenWide_03:
                "The tongue is adopted for licking and is covered with small hard spines called papillae. " +
                "The papillae act like a rasp, removing the meat from the bones of the prey." +
                "Rough tongue feels similar to the texture of sandpaper.",
            },
            part3: {
              paragraph_OpenWide_04:
                "Keeping the fur clean is an important part of a cheetah’s life. Family members spend " +
                "many hours grooming each other with their tongues. This behavior aids in the social " +
                "bonding of a cheetah group."
            }
          }
        },
        subsection_RipAndTear: {
          title: "Rip and Tear",
          contents: {
            paragraph_RipAndTear_01:
              "The cheetah’s teeth are adapted to support their eating style. By eating fast, cheetahs avoid losing their prey to other predators.",
            paragraph_RipAndTear_02:
              "The canines, “eye teeth” or “fangs”, are used for gripping and hold while the prey is being suffocated. " +
              "Cheetah’s canines are smaller and less developed than those of the lion or leopard.",
            paragraph_RipAndTear_03:
              "The incisors, “front teeth”, are used for plucking fur and skinning the carcasses. " +
              "Straight and strong incisors are essential for quick access to the meat of the prey.",
            paragraph_RipAndTear_04:
              "The carnassial, “back teeth” or “pre molars”, work in scissor like fashion and enable the cheetah to " +
              "shear large pieces of flesh which are quickly swallowed whole. When using these teeth during feeding, " +
              "cheetahs hold their heads sideways at an angle to the carcass. These blade like teeth are similar to the " +
              "lion’s and leopard’s. They do not have the same function as chewing meat as those of the jackal or crushing " +
              "bones like those of the hyenas."
          }
        }
      }
    },
  }
};
