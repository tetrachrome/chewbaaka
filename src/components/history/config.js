/**
 * history/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 07, 2020
 * Updated  : Jul 10, 2020
 */

export const config = {
  pageProps: {
    coverImage: "../history/assets/cheetah-mother-and-cubs-on-termite-mound.jpg",
    title: "History",
    subtitle: "Learn about the evolution and migration of the species, its relationship with man throughout history, and its population and ranges.",
    pageMenuItems: [
      "Evolution",
      "Cheetah and Man",
      "Range and Population"
    ],
  },
  contentPageIntro: {
    title: "History of the Cheetah",
    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
  },
  contentPageSections: {
    section_Evolution: {
      intro: {
        title: "Evolution",
        content: "The oldest fossils place  cheetahs in North America in what is now Texas, Nevada and Wyoming. Cheetahs were common throughout Asia, Africa, Europe and North America until the end of the last Ice Age, about 10,000 years ago, when massive climatic changes caused large numbers of mammals to disappear."
      },
      subsections: {
        subsection_EvolutionOfCats: {
          title: "Evolution of Cats"
        }
      }
    },
    section_CheetahAndMan: {
      intro: {
        title: "Cheetah and Man",
        content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
      },
      subsections: {
        subsection_RelationshipsWithMan: {
          title: "Relationships with Man",
          contents: {
            paragraph_01: "Try resizing the window: You'll see that each sentence is on one line when the window is wide enough. Only when the window is too narrow for the whole sentence will the sentence be broken over several lines. When you remove the 'margin-right: -50%' and resize the window again, you'll see that the sentences will be broken already when the window is still twice as wide as the text lines. ",
            paragraph_Cheetah_And_Man_01:
              'Traditional African healers and witchdoctors used cheetah foot bones in spiritualistic rituals to symbolize fleet-footnedness and speed. ' +
              'They used bones from a wide variety of animals and techniques varied from tribe to tribe. Objects ' +
              'represented a person, a thing or mood fron the past, present or future, and are known as Divinatory sets.',
            paragraph_Cheetah_And_Man_02:
              'During the time of the Egyptian pharaohs, the cheetah was considered a goddess named ' +
              '"Mafdet". Pharaohs kept cheetahs was close companions, ' +
              "symbolic of Mafdet's protection of the royal throne.",
            paragraph_Cheetah_And_Man_03:
              'The San of Southern Africa ate cheetah meat as a symbol for speed, but it was not a main food in their diet.',
            paragraph_Cheetah_And_Man_04:
              'Kings wore cheetah skins for dignity. Trade in cheetah skin only started after European explorers began ' +
              'requesting them.',
          }
        },
        subsection_CheetahsInSports: {
          title: "Cheetahs In Sports",
          contents: {
            paragraph_The_Hunting_Leopard_01:
              'In c1700 BC, the Egyptians were the first to tame cheetahs. They admired the cheetah for its speed, ' +
              'hunting ability and beauty. They honored cheetahs as symbols of royalty and prestige.',

            paragraph_The_Hunting_Leopard_02:
              'The swiftest animal on earth became a cherished hunting companion of Pharaohs and royalty ' +
              'throughout Europe, Asia and India. Until the early 1900s, ownership of cheetahs was as important ' +
              'to these nobles as their love for gold. Cheetahs hunt by sight so they excelled in the sport ' +
              'known as "coursing". Hunts organized by royalty and nobles were for the challenge of sport, ' +
              'not for food. Hunts represented power and prestige.',

            paragraph_The_Hunting_Leopard_03:
              'By the 1500s the popularity of the cheetah as a hunting companion rivalled that of the dog. ' +
              'Cheetahs, the most easily tamed of the big cats, were caught, tamed and trained. Adults were ' +
              'used because cubs had not learned how to hunt. Tamed cheetahs formed a strong bond with their keepers.',

            paragraph_The_Hunting_Leopard_04:
              'Each cheetah rode to the hunt by horseback or on a cart. Its eyes were coved with a hood and uncovered ' +
              'when prey were sighted. The cat was released to chase down the prey then rewarded with meat fed from a ' +
              'wooden spoon.',

            paragraph_The_Hunting_Leopard_05:
              'Althrough cherished, pampered cheetahs were loved to near extinction and taken from the wild in great numbers. ' +
              'By the early 1900s, India and Iran were importing African cheetahs for the sport of "coursing", as their own ' +
              'wild population became too small.'
          }
        },
        subsection_CheetahsInArt: {
          title: "Cheetahs In Art",
          contents: {
            paragraph_Cheetah_In_Art_01:
              'Sumerians (Iraqi ancestors) were the first to use cheetahs in art.',

            paragraph_Cheetah_In_Art_02:
              'The cheetah and leopard have often been mistaken for each other. ' +
              'Cheetahs in early art were frequently called "panther" ' +
              'or "hunting leopard". Early artists called the true leopard "pardus" ' +
              'and described it as a cross between a cheetah (panther), and the lion (leo).',

            paragraph_Cheetah_In_Art_03:
              'Middle Age and Renaissance artists began drawing cheetahs and other ' +
              'animals more life-like. During this time of scientific exploration, ' +
              'detailed descriptions of animals provided incentive for reality in art. ' +
              'Artists included cheetahs more often in their paintings as the trade in ' +
              'live animals increased.',

            paragraph_Cheetah_In_Art_04:
              'Artists always pictured cheetahs as animals of speed and royalty.'
          }
        }
      }
    },
    section_PopulationAndRange: {
      intro: {
        title: "Range and Population",
        content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
      },
      subsections: {
        subsection_Namibia: {
          title: "Namibia - Cheetah Capital of the World",
          contents: {
            paragraph_01:
              "Namibia has the world's largest cheetah population. " +
              "Approximately 3000 cheetahs share the land with humans, " +
              "livestock, and wildlife. " +

              "Today, the status of the Namibian cheetah is stabilizing. " +
              "During the 1980s, the population of Namibina cheetahs " +
              "declined by half. In the 10-year period, nearly " +
              "7,000 cheetahs were removed from the wild. " +

              "In most countries where cheetahs live, their numbers " +
              "have been reduced to critical levels. The cheetah's " +
              "survival worldwide is in human hands.",

            paragraph_Namibia_Home_For_The_Future:
              "The greatest hope for the cheetah's survival lies in the " +
              'pristine countryside of Namibia. Nearly 1000 Namibian farmers ' +
              "control the fate of the country's cheetahs and the land they live.",
          }
        },
        subsection_RoadToExtinction: {
          title: "The Road to Extinction ?",
          contents: {
            paragraph_The_Road_To_Extinction_01:
              "World development, industrialization, automobiles, and aeroplanes...",

            paragraph_The_Road_To_Extinction_02:
              "Through the 1900s, man's inventions seemed limitless, yet there is " +
              "nothing man-made that rivals the speed and efficiency of the cheetah.",

            paragraph_The_Road_To_Extinction_03:
              "Scientists classified the cheetah as Acinonyx jubatus, yet nobles still " +
              'referred to it as the "hunting leopard". By the end of the 1800s, cheetahs' +
              'were a rarity in Asia Minor and Arabia because of their use in the sport ' +
              'of "coursing".',

            paragraph_The_Road_To_Extinction_04:
              'Although it appeared that the cheeatahs had a large range, their numbers ' +
              'within that range existed in small pockets. Cheetahs, farmers and their livestock ' +
              'all preferred open grasslands for their habitat. Increasing agricultural development ' +
              'and new settlements played havoc with remaining cheetah populations.',

            paragraph_Where_Did_The_Cheetah_Go_01:
              'As human population grew, the amount of land devoted to livestock farming ' +
              'steadily increased. Livestock filled the open land where cheetahs roamed. ' +
              'Natural prey became scarce. Farmers kill other large predators. Although ' +
              'game reserves protected them, cheetahs could not compete against hyoenas ' +
              'and lions. Farmlands offered cheetahs a safe haven, but they sometimes killed livestock.',

            paragraph_Where_Did_The_Cheetah_Go_02:
              'Farmers saw cheetahs more frequently and thought their numbers had increased. ' +
              'Cheetahs took the blame for most predator-related livestock losses. ' +
              'Farmers killed cheetahs by the thousands as pests or to seell their skins ' +
              'to the fur trade.',

            paragraph_Where_Did_The_Cheetah_Go_03:
              'By 1975, researches realized that the cheetah was in troubles. CITES ' +
              '(Convention on International Trade of Endangered Species) placed the cheetah ' +
              'on Appendix I, making international trade in live cheetah or cheetah products illegal. ' +
              'Local laws supported CITES in many countries where cheetahs still lived. ' +
              'Researchers began looking for ways to encourage the growth of cheetah populations ' +
              'through land management practices.'
          }
        }
      }
    }
  }
};
