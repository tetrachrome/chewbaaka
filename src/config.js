/**
 * src/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 09, 2020
 * Updated  : Jul 09, 2020
 */

export const config = {
  styling: {
    mode: 'debug',
    debugStyles: {
      // 'DebugColor': true,
      'DebugWireframe': true,
    }
  }
};
