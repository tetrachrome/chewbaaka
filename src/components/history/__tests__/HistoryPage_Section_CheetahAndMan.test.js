/**
 * HistoryPage_Section_CheetahAndMan.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 10, 2020
 * Updated  : Jul 10, 2020
 */

import React from 'react';

import { render } from '@testing-library/react';

import { config } from '../config'

import renderer from 'react-test-renderer';

import HistoryPageSectionCheetahAndMan from '../HistoryPage_Section_CheetahAndMan';

test('renders HistoryPageSectionCheetahAndMan component', () => {
  render(<HistoryPageSectionCheetahAndMan config={config}/>);
});

test('HistoryPageSectionCheetahAndMan component snapshot', () => {
  const tree = renderer
    .create(
      <HistoryPageSectionCheetahAndMan config={config}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
