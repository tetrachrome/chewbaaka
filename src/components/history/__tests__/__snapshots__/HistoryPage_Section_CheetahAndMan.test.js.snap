// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryPageSectionCheetahAndMan component snapshot 1`] = `
<div>
  <div
    className="ContentPageSectionTemplateOuterContainer ContentPageSectionTemplateOuterContainer_DebugColor DebugWireframe"
  >
    <div>
      <div
        className="ContentPageSubsectionTemplateOuterContainer ContentPageSubsectionTemplateOuterContainer_DebugColor DebugWireframe"
      >
        <div
          className="ContentPageSubsectionTemplateInnerContainer ContentPageSubsectionTemplateInnerContainer_DebugColor DebugWireframe"
        >
          <h3
            className="ContentPageSubsectionTitle ContentPageSubsectionTitle_DebugColor DebugWireframe"
          />
          <div
            className="ContentPageSectionHeadOuterContainer"
          >
            <div
              data-testid="ContentPageSectionAnchorComponentTestId"
              id="cheetah-and-man"
            />
            <div
              className="ContentPageSectionHeadInnerContainer ContentPageSectionHeadInnerContainer_DebugColor DebugWireframe"
            >
              <div
                className="ContentPageSectionHeadTitleContainer ContentPageSectionHeadTitleContainer_DebugColor DebugWireframe"
              >
                <h2
                  className="ContentPageSectionHeadTitle ContentPageSectionHeadTitle_DebugColor DebugWireframe"
                >
                  Cheetah and Man
                </h2>
              </div>
              <div
                className="ContentPageIntroParagraphContentContainer"
              >
                <p
                  className="ContentPageSectionSubtitleText"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ContentPageSectionTemplateInnerContainer ContentPageSectionTemplateInnerContainer_DebugColor DebugWireframe"
    >
      <div>
        <div
          className="HistoryPageSubsectionCheetahAndManImageOuterContainer HistoryPageSubsectionCheetahAndManImageOuterContainer_DebugColor DebugWireframe"
        >
          <div
            className="ContentPageSubsectionTemplateOuterContainer ContentPageSubsectionTemplateOuterContainer_DebugColor DebugWireframe"
          >
            <div
              className="ContentPageSubsectionTemplateInnerContainer ContentPageSubsectionTemplateInnerContainer_DebugColor DebugWireframe"
            >
              <h3
                className="ContentPageSubsectionTitle ContentPageSubsectionTitle_DebugColor DebugWireframe"
              >
                Relationships with Man
              </h3>
              <div
                className="HistoryPageSubsectionCheetahAndManImageInnerContainer HistoryPageSubsectionCheetahAndManImageInnerContainer_DebugColor DebugWireframe"
              >
                <div>
                  <div
                    className="ContentPageSubsectionTwoColumnContentTemplateOuterContainer ContentPageSubsectionTwoColumnContentTemplateOuterContainer_DebugColor DebugWireframe"
                  >
                    <div
                      className="ContentPageSubsectionTwoColumnContentTemplateInnerContainer"
                    >
                      <div>
                        <div
                          className="ui vertically divided grid"
                        >
                          <div
                            className="two column row"
                          >
                            <div
                              className="column"
                            >
                              <div
                                className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer_DebugColor DebugWireframe"
                              >
                                <div
                                  className="HistoryPageSubsectionCheetahAndManImageContentTextContainer"
                                >
                                  <div
                                    className="FlexibleContainerOuterContainer FlexibleContainerOuterContainer_DebugColor DebugWireframe"
                                  >
                                    <div
                                      className="FlexibleContainerInnerContainer FlexibleContainerInnerContainer_DebugColor DebugWireframe"
                                    >
                                      <p>
                                        Traditional African healers and witchdoctors used cheetah foot bones in spiritualistic rituals to symbolize fleet-footnedness and speed. They used bones from a wide variety of animals and techniques varied from tribe to tribe. Objects represented a person, a thing or mood fron the past, present or future, and are known as Divinatory sets.
                                      </p>
                                      <p>
                                        During the time of the Egyptian pharaohs, the cheetah was considered a goddess named "Mafdet". Pharaohs kept cheetahs was close companions, symbolic of Mafdet's protection of the royal throne.
                                      </p>
                                      <p>
                                        The San of Southern Africa ate cheetah meat as a symbol for speed, but it was not a main food in their diet.
                                      </p>
                                      <p>
                                        Kings wore cheetah skins for dignity. Trade in cheetah skin only started after European explorers began requesting them.
                                      </p>
                                    </div>
                                  </div>
                                </div>
                              </div>
                            </div>
                            <div
                              className="column"
                            >
                              <div
                                className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer_DebugColor DebugWireframe"
                              >
                                <div>
                                  <div
                                    className="ImageSlidingGalleryOuterContainer"
                                  >
                                    <div
                                      className="ImageSlidingGalleryCoreContainer"
                                    >
                                      <img
                                        alt="Researchers used the Tuxtla Statuette to decipher the epi-Olmec writing system, which represents both syllables and words"
                                        className="ImageSlidingGalleryImgPart"
                                        src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/War_trophies_Deir_el_Bahari_Wellcome_L0027402.jpg/440px-War_trophies_Deir_el_Bahari_Wellcome_L0027402.jpg"
                                      />
                                      <div
                                        className="ImageSlidingGalleryCaptionPart"
                                        data-testid="ImageSlidingGalleryCaptionPart"
                                      >
                                        <div
                                          className="CaptionContainer"
                                          data-testid="CaptionComponentTestId"
                                        >
                                          <div
                                            className="IconContainer"
                                          >
                                            <i
                                              aria-hidden="true"
                                              className="image icon"
                                              onClick={[Function]}
                                            />
                                          </div>
                                          <p
                                            className="CaptionText"
                                          >
                                            Researchers used the Tuxtla Statuette to decipher the epi-Olmec writing system, which represents both syllables and words
                                          </p>
                                        </div>
                                      </div>
                                    </div>
                                    <div
                                      className="ImageSlidingGalleryButtonContainer"
                                    >
                                      <div
                                        className="ui blue huge icon left attached button"
                                        onClick={[Function]}
                                        role="button"
                                        tabIndex={0}
                                      >
                                        <i
                                          aria-hidden="true"
                                          className="arrow alternate circle left outline icon"
                                          onClick={[Function]}
                                        />
                                      </div>
                                      <div
                                        className="ui blue huge icon right attached button"
                                        onClick={[Function]}
                                        role="button"
                                        tabIndex={0}
                                      >
                                        <i
                                          aria-hidden="true"
                                          className="arrow alternate circle right outline icon"
                                          onClick={[Function]}
                                        />
                                      </div>
                                    </div>
                                  </div>
                                </div>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
